omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/ClobDatatypeStorageController.java

```diff
@@ -50,19 +50,22 @@ public String create(@RequestParam MultipartFile file, HttpServletRequest reques
 	public void retrieve(@PathVariable("uuid") String uuid, HttpServletRequest request, HttpServletResponse response)
 	        throws Exception {
 		ClobDatatypeStorage clobData = datatypeService.getClobDatatypeStorageByUuid(uuid);
-		
-		PrintWriter writer = null;
-		try {
-			writer = response.getWriter();
-			writer.print(clobData.getValue());
-			writer.flush();
-		}
-		finally {
-			if (writer != null) {
-				writer.close();
-			}
-		}
-		
+
+        if(clobData == null) {
```

Comment:
Sorry my bad. I checked the offending files and it looked like you were the last one who modified them so I automatically assumed you are the culprit :-). I guess the mess was carried forward from time immemorial :). I fixed the issues we are good. I just thought I should remind you as the culprit, apparently I got the wrong guy!